packages/services/discord.py

```diff
@@ -34,14 +35,19 @@ async def __check_commands(self, message:Message)->bool:
         commands = {
             "!commands": "",
             "!help":  help_string,
-            "!highscore":  "high score is not implemented yet"
+            "!highscore":  f"Your server highscore is: {self.db_conn.get_highscore(str(message.guild.id))}"
```

Comment:
I think I want to pull this outside of the command dict so we're only hitting the db if the command is run

Reply:
addressed in [8ebd69c](https://github.com/erindatkinson/snowball/pull/14/commits/8ebd69ce6ee68e45ab137e1b0082b8970141e00e)